README.rst

```diff
@@ -50,3 +50,27 @@ the errors and assumptions that seasoned contributors have glossed over.
 `PyCon talk <https://www.youtube.com/watch?v=6Uj746j9Heo>`_, and was adapted by
 hstdp based on its use in the README file for the
 `MetPy project <https://github.com/Unidata/MetPy>`_.*
+
+
+JupyterHub Access
+-----------------
+
+*Note:* This is currently still in research-and-development stage and is subject to change.
+
+To run a pre-installed pipeline in JupyterHub:
+
+* Click on https://dev.science.stsci.edu/hub/spawn?image=793754315137.dkr.ecr.us-east-1.amazonaws.com/datb-tc-pipeline-nb:hstdp-snapshot and sign in.
+* Click "Terminal" to:
```

Comment:
I don't know why this line renders as bold in the preview. GitHub doesn't play well with RST sometimes. :woman_shrugging: